modules/appeals_api/app/controllers/appeals_api/v1/appeals_controller.rb

```diff
@@ -2,15 +2,77 @@
 
 module AppealsApi
   module V1
-    class AppealsController < AppealsApi::V0::AppealsController
+    class AppealsController < AppealsApi::ApplicationController
+      include AppealsApi::CaseflowRequest
       include AppealsApi::OpenidAuth
 
+      skip_before_action :authenticate
+      before_action :validate_icn_parameter, only: %i[index]
+
+      ICN_REGEX = /^[0-9]{10}V[0-9]{6}$/
       OAUTH_SCOPES = {
         GET: %w[veteran/AppealsStatus.read representative/AppealsStatus.read system/AppealsStatus.read]
       }.freeze
 
+      def index
+        render json: caseflow_response.body, status: caseflow_response.status
+      end
+
       private
 
+      def required_header(name)
+        value = request.headers[name]
+        raise(Common::Exceptions::BadRequest, detail: "Header '#{name}' is required") if value.blank?
+
+        value
+      end
+
+      def ssn
+        @ssn ||= icn_to_ssn!(params[:icn])
+      end
+
+      def va_user
+        required_header('X-VA-User')
+      end
+
+      def caseflow_request_headers
+        { 'Consumer' => request.headers['X-Consumer-Username'], 'VA-User' => required_header('X-VA-User') }
```

Comment:
I haven't updated `X-VA-User` or `X-Consumer-Username` at all here - if either these should be in params instead, I can make another update here or make a follow-up PR instead